.github/workflows/unit-tests-refresh.yml

```diff
@@ -24,19 +24,10 @@ jobs:
         timeout-minutes: 10
         run: containers/unit-tests/build
 
-      - name: Run pytest-cov test
-        timeout-minutes: 15
-        run: containers/unit-tests/start --verbose --env=CC=gcc --image-tag=latest --make pytest-cov
-
       - name: Run amd64 gcc check-memory test
         timeout-minutes: 20
         run: containers/unit-tests/start --verbose --env=CC=gcc --image-tag=latest --make check-memory
 
-      - name: Run i386 clang check-memory test
-        timeout-minutes: 20
-        # HACK: https://bugs.kde.org/show_bug.cgi?id=452758
```

Comment:
Oh, this was i386 only! I checked and thought it would be clang 14 issue and as Debian wasn't updated yet to something newer we couldn't remove it. But seems I'm wrong \o/